### PR TITLE
Update UART1 whitelist for usability.

### PR DESCRIPTION
**Pull request by denniszollo**
My task from triage meeting.

Requires console testing and SBP2RINEX testing before merging; both should work with a file recorded over UART1.

Will break HITL since HITL probably only uses the ECEF messages and is connected over UART1. @axlan

### CODE REVIEW

File: package/piksi_system_daemon/src/ports.c [hunk 1]
```diff
@@ -88,7 +88,7 @@ static port_config_t port_configs[] = {
   },
   {
     .name = "uart1",
-    .opts = "--file /dev/ttyPS1 --nonblock --outq 4096",
+    .opts = "--file /dev/ttyPS1 --nonblock --outq 4096 --debug",
```
Comment (mfine) [inline package/piksi_system_daemon/src/ports.c]: Do we want a `--debug` flag checked in?

**Comment by anth-cole** [comment]
lgtm - I'm nervous about checking in the `--debug` flag too, it seems like this could cause problems and maybe we should do this on master only.

**Comment by denniszollo** [comment]
I think we will have to remove --debug flag before release.  

I also now propose that we double size of the serial out buffer queues so that in the worst case they hold 0.57 seconds worth of data rather than 0.28.

**Comment by denniszollo** [comment]
I removed --debug flag (since it didn't seem to do much) and obs decimation.  I also tested extensively and feel good about it.

**Pull request merged**